src/odemis/acq/align/autofocus.py

```diff
@@ -249,6 +249,7 @@ def _DoBinaryFocus(future, detector, emt, focus, dfbkg, good_focus, rng_focus):
     returns:
         (float): Focus position (m)
         (float): Focus level
+        (float): Focus confidence (0<=f<=1)
```

Comment:
Could you add that 0 means it's certainly not in focus, and 1 means it's certainly the best focus possible.